API/DEPR: Change default skipna behaviour + deprecate numeric_only in Categorical.min and max (#27929)
pandas-dev/pandas: doc/source/whatsnew/v1.0.0.rst

   df.groupby(["cat_1", "cat_2"], observed=False)["value"].count()

By default :meth:`Categorical.min` return the min and the max respectively
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

When :class:`Categorical` contains ``np.nan``,

> :meth:`Categorical.min` no longer return ``np.nan`` by default (skipna=True).

Reviewer: can you add the issue number here

> When :class:`Categorical` contains ``np.nan``, :meth:`Categorical.min` and :meth:`Categorical.max`

Reviewer 2: ```suggestion
When :class:`Categorical` contains ``np.nan``, :meth:`Categorical.min`
```

It was only `min` that returned NaN (can you update the title as well?)

> no longer return ``np.nan`` by default.

Reviewer 2: Maybe add something like "to honor the default of `skipna=True`" to make it clear that this change makes it consistent with the rest of pandas

   df.groupby(["cat_1", "cat_2"], observed=False)["value"].count()

> By default :meth:`Categorical.min` return the min and the max respectively

Reviewer 2: ```suggestion
By default :meth:`Categorical.min` now returns the minimum instead of np.nan
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

When :class:`Categorical` contains ``np.nan``,

> :meth:`Categorical.min` no longer return ``np.nan`` by default (skipna=True) (:issue:`25303`)

Reviewer 2: ```suggestion
:meth:`Categorical.min` no longer return ``np.nan`` by default (to honor the ``skipna=True`` default) (:issue:`25303`)
```
